learn/getting-started.md

```diff
@@ -257,4 +257,29 @@ This logic can be overridden by defining a custom ServiceVersionResolver
     };
      
     // apply resolver to client 
-    OSClient.withConfig(Config.newConfig().withResolver(resolver)) 
\ No newline at end of file
+    OSClient.withConfig(Config.newConfig().withResolver(resolver)) 
+
+#### Endpoint URL Resolver ####
+
+Resolving and endpoint URL is by default based on the Service Type and Facing perspecitve. This logic can be changed by overriding a custom EndpointURLResolver
```

Comment:
Resolving an endpoint [..] perspective.
The default logic can be overridden by a [..] .